app/assets/javascripts/templateFolderForm.js

```diff
@@ -147,7 +147,7 @@
       );
 
       // use dialog mode for states which contain more than one form control
-      if ($.inArray(this.currentState, ['move-to-existing-folder', 'add-new-template'])) {
+      if ($.inArray(this.currentState, ['move-to-existing-folder', 'add-new-template']) !== -1) {
```

Comment:
Is there any benefit to the jQuery in that case?

```suggestion
      if (['move-to-existing-folder', 'add-new-template'].indexOf(this.currentState) !== -1) {
```

Reply:
Not if our version of babel polyfills `Array.indexOf`. I'll check...